submission/distributor_test.go

```diff
@@ -126,7 +126,7 @@ func sampleUncollectableLogList() *loglist3.LogList {
 	ll.Operators[0].Logs = append(ll.Operators[0].Logs, &loglist3.Log{
 		Description: "Does not return roots", Key: []byte("VW5jb2xsZWN0YWJsZUxvZ0xpc3Q="),
 		URL:   "uncollectable-roots/log/",
-		DNS:   "uncollectavle.ct.googleapis.com",
+		DNS:   "uncollectable.ct.googleapis.com",
```

Comment:
It's possible this was intentional, but I'm hoping it isn't and doesn't matter...